.circleci/cimodel/data/simple/binary_smoketest.py

```diff
@@ -153,7 +160,17 @@ def gen_tree(self):
             "executor": "windows-with-nvidia-gpu",
         },
     ),
-
+    SmoketestJob(
```

Comment:
Actually we have smoke test jobs for nightly packages, so this is just for testing, right?

Reply:
Yes, what I want is to make sure my PR in builder can run properly.
And I can get the [verified link](https://app.circleci.com/pipelines/github/pytorch/pytorch/235594/workflows/ad93d2da-ccba-4dad-b835-fd1d6b3023f5/jobs/8725700) easily